charts/metering-ci/templates/metering.yaml

```diff
@@ -63,6 +63,13 @@ spec:
         # or less
         prometheusMetricsImporterStepSize: "60s"
 
+      {{- if or .Values.reportingOperatorMemory .Values.reportingOperatorCpu }}
```

Comment:
Does it matter where this line is aligned? Seems like the CI metering file is pretty inconsistent in that regard.

Reply:
At the same time, it's the CI metering file, so it's probably not something that needs to be worried about.

Reply:
No, because of the `-` in the `{{-`, but I'm going to fix this up. Indenting a bit helps since it's hard to find which if/ends go together.